src/gluonts/dataset/pandas.py

```diff
@@ -93,24 +80,21 @@ class PandasDataset:
         Dict[str, pd.Series],
     ]
     target: Union[str, List[str]] = "target"
+    feat_dynamic_real: Optional[List[str]] = None
+    past_feat_dynamic_real: Optional[List[str]] = None
     timestamp: Optional[str] = None
     freq: Optional[str] = None
-    feat_dynamic_real: List[str] = field(default_factory=list)
-    feat_dynamic_cat: List[str] = field(default_factory=list)
-    feat_static_real: List[str] = field(default_factory=list)
-    feat_static_cat: List[str] = field(default_factory=list)
-    past_feat_dynamic_real: List[str] = field(default_factory=list)
+    static_features: Optional[pd.DataFrame] = None
     ignore_last_n_targets: int = 0
     unchecked: bool = False
     assume_sorted: bool = False
     _pairs: Iterable[Tuple[Any, Union[pd.Series, pd.DataFrame]]] = field(
         init=False
     )
+    _static_reals: Optional[pd.DataFrame] = None
+    _static_cats: Optional[pd.DataFrame] = None
```

Comment:
If they are not meant to be part of the arguments, let's use `= field(init=False)`.

Reply:
ah! I somehow forgot to do that